### ISSUE TITLE
Using: Leaflet.MarkerCluster

### ISSUE THREAD

**Issue by RobertReynolds1** [ramnathv/rMaps#71]
Hi Ramnath,

rMaps is an incredible package! Many thanks for putting it together.

I'm very new to using rMaps so apologies for the very basic question. Is it possible to use the Leaflet.MarkerCluster plugin (http://leafletjs.com/2012/08/20/guest-post-markerclusterer-0-1-released.html) with rMaps? By the way, I have seen the heat maps http://rmaps.github.io/blog/posts/leaflet-heat-maps/index.html (which are great!).

Many thanks,
RR

**Comment by imlearningr** [comment]
yes rMaps is such an awesome package! thank you so much Ramnath for all creating it and sharing it with us! 
Robert and Ramnath , I am trying to accomplish the exact same thing with the Leaflet.MarkerCluster plugin. I followed the great demonstration of the heat maps and it was so cool, But I don't have enough r or java knowledge to pull it off for the MarkerCluster plugin on my own, I am wondering whether you have found a solution. I would really appreciate any thought or suggestion. thank you so much!
